src/main/java/uk/gov/companieshouse/model/dto/companyOfficers/CompanyOfficer.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.companieshouse.model.dto.companyOfficers;
+package uk.gov.companieshouse.model.dto.companyofficers;
```

Comment:
hmmm the filename in GitHub is still showing as companyOfficers instead of companyofficers.

I believe unit tests and usages via imports of these classes will also need to be updated.